builder/dockerfile/instructions/parse.go

```diff
@@ -271,16 +272,17 @@ func parseFrom(req parseRequest) (*Stage, error) {
 		return nil, err
 	}
 
+	flPlatform := req.flags.AddString("platform", "")
```

Comment:
Nit: I am not a fan of this Hungarian-style notation, preferring `platform` or `platformFlag`. I realize there is a precedent for the `fl` prefix in the builder code, so it's okay, for consistency, but I think it should be cleaned up eventually. :)